test/model/card/action/simple_cost_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:treasure_of_the_high_seas/model/card/action/simple_cost.dart';
+import 'package:treasure_of_the_high_seas/model/game_state_factory.dart';
+import 'package:treasure_of_the_high_seas/model/hand.dart';
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+main() {
+  test('should correctly report whether it is affordable', () {
+    final cost = SimpleCost([Resource.FOOD]);
+
+    expect(cost.isAffordable(Hand([])), false);
+    expect(cost.isAffordable(Hand([Resource.CREW])), false);
+    expect(cost.isAffordable(Hand([Resource.FOOD])), true);
+  });
+
+  test('should describe its cost', () {
+    expect(SimpleCost([Resource.FOOD]).getDescription(), "(F)");
+    expect(SimpleCost([Resource.FOOD, Resource.FOOD, Resource.LANDLUBBER]).getDescription(), "(F, F, L)");
+  });
+
+  test('should correctly report when its empty', () {
```

Comment:
It's*

Reply:
can't do the apostrophe as we're in single quotes already. Will change to `it is` :stuck_out_tongue: